apps/i18n/common/en_us.json

```diff
@@ -1245,6 +1245,7 @@
   "joinText": "join",
   "joinTextTooltip": "Create a piece of text by joining together multiple items.",
   "joinUs": "Join us",
+  "jsonInCDOBlockly": "Due to a system upgrade, your code could not be loaded. Please use Version History to recover a working version of this project, or to start over.",
```

Comment:
nit: Thoughts on CDOBlockly vs. CdoBlockly? I thought it was CdoBlockly everywhere, and it looks like it is most places, except for where we `import CDOBlockly from '@code-dot-org/blockly'`?

Reply:
I don't have a strong preference, if in most places it's `Cdo` I will go with that